presto-orc/src/main/java/com/facebook/presto/orc/metadata/EncryptionGroup.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.metadata;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.slice.Slice;
+
+import java.util.List;
+import java.util.Optional;
+
+public class EncryptionGroup
+{
+    // Sub trees in the schema that encryption should be applied
+    // Sub tree is identified by its root id
+    private final List<Integer> nodes;
+    // Arbitrary binary representing key metadata. It could be identifier
+    // of key in KMS, encrypted DEK or other form of user defined key metadata.
+    //  this is optional and can use first keyMetadata from stripe
+    private final Optional<Slice> keyMetadata;
```

Comment:
keyMetadata and statistics are not used? Why is that?

Reply:
the key is the one that encrypts the statistics.  We don't use statistics because of https://github.com/prestodb/presto/blob/561e977f3b39c056ef76ea00ff826f0148039b93/presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java#L98